pkg/apis/application/v1alpha1/types.go

```diff
@@ -629,6 +629,8 @@ const (
 	ApplicationConditionRepeatedResourceWarning = "RepeatedResourceWarning"
 	// ApplicationConditionExcludedResourceWarning indicates that application has resource which is configured to be excluded
 	ApplicationConditionExcludedResourceWarning = "ExcludedResourceWarning"
+	// ApplicationConditionOrphanedResourceWarning indicates that application has orphaned resources
```

Comment:
do you need to state this is only populated if configured on?

Reply:
covered in docs